blogposts/2022/release-3.38.md

```diff
@@ -0,0 +1,77 @@
+---
+title: 'Sourcegraph 3.38 release'
+publishDate: 2022-03-21T10:00-07:00
+description: 'Sourcegraph 3.38 introduces improved Notebooks, faster Code Intelligence for large repos, dependencies search, and custom file syntax highlighting.'
+tags: [blog, release]
+slug: 'release/3.38'
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.38/sourcegraph-3-38-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.38/sourcegraph-3-38-release.png
+changelogItems:
+  - description: Code Insights will now periodically clean up data series that are not in use. There is a 1 hour grace period where the series can be reattached to a view, after which all of the time series data and metadata will be deleted.
```

Comment:
@Joelkw this is missing what the benefit/impact is. It describes only what happens.

Reply:
```suggestion
  - description: Code Insights will now periodically clean up data series that are not in use. There is a 1 hour grace period where the series can be reattached to a view, after which all of the time series data and metadata will be deleted. This means customers deploying their own instance won't be storing data that has no use.
```

Reply:
Something like this? We can also delete if we don't want to include – this all just gets taken from a script of the changelog updates.